docs/reference/docs/reindex.asciidoc

```diff
@@ -499,6 +520,7 @@ POST _reindex?pretty
 }
 --------------------------------------------------
 // AUTOSENSE
+// TEST[continue]
```

Comment:
I think it is kind of cool how the doc gives an example with multiple snippets and it is super natural to just join them together into a single test.